### PR TITLE
disabling weather station look up during site refresh

### PR DESCRIPTION
**Pull request by Baalmart**
# temporarily disabling weather station look up during site refresh

**_WHAT DOES THIS PR DO?_**
disabling weather station look up during site refresh, due a bad gateway timeout.

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
[PLAT-1047]

[PLAT-1047]: https://airqoteam.atlassian.net/browse/PLAT-1047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/device-registry/utils/create-site.js [hunk 1]
```diff
@@ -801,59 +794,52 @@ const manageSite = {
...
-        request["body"]["nearest_tahmo_station"] = nearest_tahmo_station;
-      }
-
-      if (responseFromNearestWeatherStation.success === false) {
-        logObject(
-          "unable to find the nearest weather station",
-          responseFromNearestWeatherStation
-        );
-      }
+      // const responseFromNearestWeatherStation = await manageSite.findNearestWeatherStation(
```
Comment (Baalmart) [inline src/device-registry/utils/create-site.js:807]: just commented out section for getting the nearest weather station. Shall reenable in future PR after refactoring.

Comment (Baalmart) [reply src/device-registry/utils/create-site.js:807]: It was causing a 504 error of the sites refresh endpoint. There is a problem with the Tahmo weather API.

Comment (NoahNsimbe) [reply src/device-registry/utils/create-site.js:807]: Is it possible to set a request timeout Instead of commenting out the code @Baalmart

Comment (Baalmart) [reply src/device-registry/utils/create-site.js:807]: Exactly!
That is what I will look into.... in a future PR as hinted in the comment section earlier. That is: "handling request time outs with the Axios client" @NoahNsimbe

**Review by NoahNsimbe — APPROVED** [review]

**Pull request merged**